\chapter{Introduction}
\vspace{-1em}
\epigraph{
  \textit{In the drama of life on a molecular scale, proteins are where the action is.}
}{Lesk\cite{lesk_introduction_2001}}
%
In contrast to other important biological molecules, proteins have a remarkable diversity of spatial structures and thus a wide variety of biological functions. Some of the many complex tasks proteins can perform include the ability to\cite{lodish_molecular_2004, echenique_introduction_2007}
\begin{itemize}
\item interpret inter-cellular signals such as insulin \& EGF in signaling pathways\cite{ghosh2006simulation, miller2007systems}
\item transport other species\cite{zakharov2009microrheological}
\item govern chemical conversion
\item control gene expression
\item convert chemical energy into mechanical energy
\item serve as building blocks to biological structures (\ie collagen and virus coats) and,
\item act as chaperones to help other biological agents function (such as GroEL in bacteria).
\end{itemize}
As such, the study of proteins, either directly or indirectly, encompasses the core of many microbiological studies. A protein's ultimate function and activity is determined by a process called folding. This folding process guides the protein into a specific three-dimensional form. Unsurprisingly, protein malfunction due to misfolding is the culprit of many clinical disorders, from cancers to abnormal protein aggregations (leading to neurodegenerative disorders like Huntington and Alzheimer\cite{uversky_intrinsic_2009}).

The study of proteins is paramount to the understanding of life itself. In this thesis we attempt to elucidate some of the properties associated with the folding process. We present a detailed study of the conformational states under various conditions.

\section{Protein Structure}

A protein can be broken down into a hierarchy of structures, each with a different set of motifs describing it. A protein is a polymer linking together a series of amino acids. Like DNA, a protein is composed of a simple alphabet of twenty naturally occurring amino acids (whereas DNA has a four-letter alphabet of base pairs). These amino acids are connected along a backbone of carbon, nitrogen and oxygen atoms. A protein has a primary structure enumerated by the ordered sequence of these letters. These encodings, along with the physiological conditions, uniquely determine the structure of the protein and hence its ultimate function. 

Amino acids connected along the backbone cannot assume arbitrary angles with respect to each other.  This stereochemistry is vividly displayed on Ramachandran plots.\cite{ramachandran_stereochemistry_1963} These plots typically illustrate the dihedral (or $\phi$-$\psi$) angles that are allowed due to energetic considerations. This is just one type of the forces which guide the folding process. 

At this structural level, certain motifs are seen more often than others. The most frequent structures common to proteins are the $\alpha$-helices and the $\beta$-strands. At this level of hierarchical complexity, the motifs are called the secondary structure. At the next level up, the tertiary structure involves the arrangement of the secondary-structure elements, helix bundles, and $\beta$-sheets. These are in turn coalesced into even larger quaternary structures for proteins with multi-domains.

The secondary structures are a good starting point for the study of the protein. In the absence of stabilizing interactions, the protein will be unfolded in a so-called random-coil structure. When hydrogen bonds form between residues along the backbone, the protein will form well-defined (locally periodic) structures of $\beta$-sheets and $\alpha$-helices. Hydrogen-bonds, hydrophobic interactions, van der Walls and electrostatic interactions all contribute to the folded structure of a protein.

\subsubsection{Secondary Structures: 
  \texorpdfstring{$\alpha$}{Alpha}-helices and 
  \texorpdfstring{$\beta$}{Beta}-sheets 
}

The first of the secondary structures, the $\alpha$-helix, is stabilized by hydrogen bonds between the \chem{CO} and \chem{HN} groups of the backbone four residues down the chain. In theory, there are multiple ways a helix could be formed: both right-handed, left-handed and different `tightness' of the helix. Almost all of these other conformations except for the $\alpha$-helix are energetically unfavorable and are rarely observed in real proteins. In an $\alpha$-helix all the hydrogen-bonds are made by neighbors that are close along the chain, thus the stabilizing forces involved are often referred to as short-range. These helices are formed very rapidly, on the order of tenths of $\mu s$. There are approximately 3.6 residues per helical turn, which from a modeling standpoint, presents a problem if the protein is coarse-grained onto a lattice that can not accommodate the exact structure.

The $\beta$-sheet is a lateral packing of $\beta$-strands; each strand is a short extended peptide segment. Just like the $\alpha$-helix, a $\beta$-sheet can exist in several different forms, however these other forms are actually found in real proteins. The two main distinctions are parallel and anti-parallel $\beta$-sheets. The interactions between the $\beta$-stands act over short distances but can be far apart along the length of the chain; they are often referred to as long(er)-range forces when compared to the $\alpha$-helix. Again, in contrast to $\alpha$-helices, $\beta$-sheets  have a wide variability in folding times. Some shorter peptide sequences can form as quickly as $0.5\mu s$, while others can take hours!\cite{xu_probing_2008} 

\section{Protein folding}
Since Anfinsen's discovery in the early 1960's that a denatured protein can spontaneously self-assemble when denaturants are removed,\cite{anfinsen_kinetics_1961} researchers have attempted to deduce which mechanisms and pathways are important to the folding process. One of the main challenges of the field is to predict the final native structure from the primary structure. To put it mildly, this has not been an easy task. Not only does a peptide (small protein) contain hundreds of atoms, it resides in a water based solvent where the effects are mediated out at lengths of several molecules away. A typical molecular dynamics simulation contains thousands of atoms making a full scale quantum computation impossible by todayâ€™s standards. Classical kinetic approximations are possible, yet consume thousands of hours of computing time for a single trajectory. This is tantamount to a limited observational study; predictions of statistical quantities such as specific heat can only be crudely estimated. These molecular dynamics simulations are, however, the best method for studying dynamics of the system without the need for costly experiments. The mechanism in which protein folding occurs can often be determined out of such \textit{in silico} experiments. 

Backing up the veracity of the computational molecular experiments are the physical experiments themselves. The traditional methods of studying an unknown sample such as mass spectrometry, lacked the resolution needed when dealing with complexities of proteins. The more detailed studies of protein structure came from advances in experimental instrumentation. These experimental techniques were sometimes adapted from other fields, but often were invented for the study of biological molecules. X-ray crystallography was created to study the periodic arrangements in crystals but later adapted to study the atomic arrangements of protein structures (for those that could be crystallized). Over the last fifty years, there have been unprecedented experimental advances such as nuclear magnetic resonance (NMR), F\"{o}rster resonance energy transfer (FRET), circular-dichroism, optical tweezers and atomic force microscopy. All of these techniques contribute to our understanding of the folding process both \textit{in vitro} and in the larger biophysical context.

\subsection{How can proteins be simplified?}

One of the reasons the four levels of structure nomenclatures exist is the inherent simplifications each level implies.  For example, at the secondary structure level most $\alpha$-helices can be expected to have the same energetic and entropic properties. Between the primary structure, the one-dimensional description of the protein as a linear chain of amino acids, and the three-dimensional conformation, there are a host of intermediate coarse-grained models. There is strong motivation for doing such a coarse graining. In a large study of the relevant interaction strengths for each of the twenty amino acids (written in terms of the so-called MJ matrix\cite{miyazawa_estimation_1985}) the resulting basis set is smaller than twenty. In other words, by diagonalizing the MJ matrix one finds two dominant eigenvectors and three large but slightly smaller eigenvectors. The remaining eigenvalues associated with the other eigenvectors are an order of magnitude smaller than these dominant eigenvectors. This implies that there is a basis for the coarse-graining of the interactions between residues. Indeed, a good approximation to the interactions found in a real protein would be the reduction to only two types, hydrophobic and hydrophilic!\cite{dill_theory_1985} The remaining interactions can be roughly classified as those belonging to the charged and uncharged types.


\subsection{Hydrophobicity}

Hydrophobicity, literally the fear of water in Greek, plays a central role in protein folding.  It was quickly determined that one of the most dominant factors in the folding process was the interaction of certain residues with the solvent. \chem{H_2 O} is an exceptional molecule with an unusually high specific heat compared to other liquids. In addition, its polar nature makes for a non-isotropic fluid with regards to the orientation of the molecules. Residues such as alanine, valine, leucine, isoleucine, phenylalanine, tryptophan and methionine are often found buried in the core of the native state conformation, while the other charged and polar residues are exposed to the solvent.\cite{pace_forces_1996} When water encounters a solute it has been known to build hydrogen-bond networks around it.\cite{lodish_molecular_2004} In the case of a protein, it is not altogether clear however if these hydrogen networks are stabilizing or destabilizing. Though early literature often thought it was one of the sole driving forces for folding, depending on the hydrophobicity of a residue it may or may not become part of the hydrogen-bond network. For a comprehensive review of the arguments see Rose.\cite{rose_hydrogen_1993}

From an entropic perspective, this presents a facet to the folding problem. A protein will try to minimize the exposure of its hydrophobic amino acids to the solvent. As a first-order effect, hydrophobic resides serve to minimize the accessible surface area.

\section{Entropy and The Science of Counting}

Entropy has a reputation for being an abstract physical concept, mathematically defined as the logarithm of density of states. We will see that a clear intuitive interpretation can be developed using the canonical example of a simple coin. When we flip a coin, the particular trajectory of the flight is irrelevant for the purposes of long-term averages. What matters is the final outcome, heads up or  heads down. With one throw, we know very little about the coin, in fact we can't even say with certainty that it will ever land on the other side! However, one of the key assumptions of statistical mechanics (the framework on which many physical ideas of entropy are based) is that of a large sample size.  So instead of throwing the coin once, we throw the coin millions\footnote{Depending on the type of systems studied, millions can be a gross underestimate. For example, typical calculations that involve macroscopic quantities can easily have $2^{\chem{N_A}}$ (where $\chem{N_A} \approx 10^{23}$) possible states!} of times. Alternatively the conclusions made would be the same if we observed one throw from a million similar coins. Once we've observed this great multitude of throws and tally the results, we can estimate the averages of the system. We call this average the density of states (for the simple coin, it is a count of the frequency of heads and tails). In almost all contexts, the usefulness of the density of states is the same as if it were given in relative proportions. For example instead of our sample experiment with say, 300,000 heads up and 700,000 heads down we could express the density of states as $\{ g_{\text{heads}}=0.3$,  $g_{\text{tails}}=0.7 \}$. Entropy then, is the study of the way a system can arrange itself. In short, an entropic theory is a combinatorial one. The formal connection of entropy to statistical mechanics will be derived in detail in Chapter \ref{chap:comp_methods}.

The underlying subject of this thesis is the study of entropic effects on the protein folding process. In a protein many factors come into play, both energetic and entropic. Entropy may not be the dominant factor in the folding process. In the spirit of teasing out the first principles of protein folding we try to study the entropic effect exclusively. Regardless of the potential, there exist disallowed regions in the conformational space that can be approximated as entropic forces. There are several energetic contributions that effect the free energy of a folded chain. If these effects are simplified to all or nothing (\ie the black spots on a Ramachandran plot) then they become purely entropic effects. The `excluded-volume' effect arising from any non-specific hard-core repulsion is another common example of this entropic force. 

\subsection{Levinthal's paradox}

It was suggested by Levinthal in 1969 that, by any reasonable measure, the state space for a protein is enormous. There exists however, a unique native state among them which makes folding impossible if the protein were to blindly sample the states.\cite{levinthal_are_1968} Consider the simplest model, where each amino acid is either in the folded or unfolded states (Levinthal used a more complicated construction, but the result remains the same). In a modest peptide of fifty amino acids this gives $2^{50}$ different possible microstates. Even if the peptide were to sample $10^{11}$ of these microstates each second\footnote{The time scale for $\phi$-$\psi$ rotations is on order of $10^{-11}$s} it still would take an impossible amount of time to reach the native conformation. If a protein were to blindly sample the state space this argument might be valid. However, it is very clear (both from experiment, and the mere existence of our biological function) that the folding rate for proteins is much faster than initially suggested by Levinthal's paradox.

It is clear from his original paper where this \textit{gedanken} was proposed that Levinthal did not see this as a paradox at all. In fact, the experimental contradiction to the paradox is what led to a fundamental idea behind protein folding, that of the funneled energy landscape. Levinthal proposed that proteins fold cooperatively, that is, the spontaneous folding happened as a result of guided interactions, much like a rock tumbling down a hill. 

This poses an interesting dilemma for the entropic study of protein folding. On one hand, the idea of a funneled energy landscape suggests that a folding pathway captures the essential dynamics of the process. However, it has been shown that landscape itself is not a smooth function, rather it is full of intermediate potential wells and entropic plains that can trap the folding process. As the physiological system's parameters change, such as the temperature or colsolute packing fraction of the system, these intermediates become more numerous and important to the folding process.

\subsection{Macromolecular Crowding}

Observational studies on protein folding have long ignored the true cellular environment, mainly to isolate the effects of a specific interaction or force. However, high concentrations of macromolecules serve to reduce the accessible volume available to the protein folding process. The interactions between the proteins and the crowding agents may be energetic but one often studies the general crowding effect alone due to the non-specific interactions. The  reduction in conformational state space then becomes a purely entropic interaction whose contribution has various approximations, such as scaled-particle theory and several integral field theories. By simply confining the protein to hard boundary conditions one can crudely estimate the crowding effect,\cite{mittal_thermodynamics_2008, wang_confinement_2009, zhou_stabilization_2001} though we have investigated the changes in the conformal space directly using a lattice model and an aspherical scaled-particle theory.\cite{hoppe_protein_2011}

\section{Chapter Overview}

The thesis is broken into four major research chapters, each exploring a different aspect of entropy in the context of protein folding: entropic forces (Ch. \ref{chap:entropic_force}), conformational states under crowding (Ch. \ref{chap:WL_crowding}), aggregation (Ch. \ref{chap:potts_aggregation}), and macrostate kinetics from microstate trajectories (Ch. \ref{chap:clustering_kinetics}). Along the way we will discuss the implications that our models have on the topics of crowding and protein aggregation. The studies look separately at the solvent, macroscopic crowders and the protein folding pathway itself. A brief summary of each research chapter is given below. In addition, an introduction to the analytical and computational methods is given (Ch. \ref{chap:comp_methods}).

\subsubsection{Entropic forces and Cosolute Flows}
In Chapter \ref{chap:entropic_force}, we consider a simplified treatment of a cosolvent (crowders) acting on a protein. What sets this study apart from others is the prediction of a stabilizing entropic force that arises purely from the \textit{motion} of the cosolvent. It has been long known that an irregular cavity in a fluid will produce an anisotropic distribution of a cosolvent, but we consider under Newtonian assumptions, the averaged velocity profiles as well. The effect of these profiles shows that, in the absence of other effects, irregular conformations are less preferred than their more compact counterparts.

The study of the excluded-volume effects on protein stability and reactions or the stability of colloidal suspensions is an active area of research. Using hard-disc collisional dynamics we investigate whether the presence of a crowding agent can induce a shape change from a non-spherical molecule to a spherical one. We show the averaged density profiles and velocity field of hard-disc crowders with an interior non-circular convex shape as a boundary condition. The density profile is not axially symmetric, consistent with other hard-potential experiments with asymmetry. However, the averaged velocity field was found to have a non-zero curl, implying a region of vorticity without a thermal gradient, advective field or other motivating potential. To explain the occurrence of the vortices, a theoretical model is provided based on the conservation of angular momentum of hard discs at contact. All these results, as well as difference in pressure along the axes, support the fact that as the packing fraction of the crowder rises, increasing force is exerted on an asymmetric molecule toward a symmetric one.

\subsubsection{Conformational States Under Crowding}
In Chapter \ref{chap:WL_crowding} we introduce the idea of the implicit crowding method to study the statistical mechanical behaviors of folding of $\beta$-sheet peptides. Using a simple bead-lattice model we are able to consider, separately, the conformational entropy involving the bond angles along the backbone and the orientational entropy associated with the dihedral angles. We use an Ising-like model to partially account for the dihedral angle entropy and implicitly, the hydrogen-bond formations. We also compare our results to recent experiments and find good quantitative agreement on the predicted folded fraction. Based on the predictions from the scaled particle theory we investigate changes in the melting temperature of the protein, suggesting crowding enhanced stability for a variant of trpzip hairpin and a slight instability for the larger $\beta$-sheet designed peptides.

\subsubsection{Potts Aggregation Models}
In Chapter \ref{chap:potts_aggregation} we examine the protein at a different level of complexity. We move from the scale of a single protein to that of an aggregated species through several models. The study is primarily motivated by disorders thought to be caused by aggregated species, such as Alzheimer's disease. We consider a simplified first-order model of cluster (oligomer) growth. The model is sufficient only as coarse approximation, so we develop a methodology to solve a general graph oriented aggregation model. In this chapter we present a new operator expansion method to solve the Ising/Potts model with external fields over an arbitrary graph. This method allows us to present new results on the Potts model problem: a general expansion of a one-dimensional lattice along with suggestions to a generalization of higher dimension. In addition, we solve a wider class of problems with a subgraph recursion relation, with the focus on lattice strips, graphs that are finite in one direction and grow in another.

\subsubsection{Clustering and Kinetics}
In Chapter \ref{chap:clustering_kinetics} we extract macrostate information from the density of states and time series data. We develop a method to cluster the microstates into physically meaningful macrostates. The conformations are grouped by similar relaxation times from a transition matrix.  The method is applied to two very different systems, a frustrated Langevin walk and a lattice model of a $\beta$-hairpin. In the random walk we are able to reduce the conformational state space from a continuous two-dimensional potential to a simple linear model with four discrete states. In the $\beta$-hairpin model, we are able to extract the folding pathway and show that the defining kinetic pathway is the formation of the turn on the hairpin.
